src/org/labkey/test/ClientErrorAssertionError.java

```diff
@@ -0,0 +1,9 @@
+package org.labkey.test;
+
+public class ClientErrorAssertionError extends AssertionError
```

Comment:
Having `Error` in the class name makes sense but reads strangely. A line of JavaDoc on intended use here and in the server-side equivalent would be nice.